guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Resolving_Package_Dependencies_{context}"]
+= Resolving Package Dependencies
+
+In {Project}, you can use the package dependency resolution feature to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
+
```

Comment:
```suggestion
```

If you keep this empty line, each sentence would stand of itself. IMHO a general rule is that a paragraph consists of at least two sentences.